packages/client/src/workflow-client.ts

```diff
@@ -1308,6 +1309,28 @@ export class WorkflowClient extends BaseClient {
     };
   }
 
+  /**
+   * Return workflow execution count by given `query`.
```

Comment:
I see you copied the comment from `list()` above, but it might not have been that great of an example to follow. Let's try to improve this doc a bit (and the one above, while we're at it). Maybe start with the suggestion below.

```suggestion
   * Return the number of Workflow Executions matching the given `query`. If no `query` is provided, then return the total number of Workflow Executions for this namespace.
```

We should also point out clearly that counts are approximates and eventually consistent.